spec/unit/interface/option_spec.rb

```diff
@@ -15,6 +15,11 @@
       end
     end
 
+    it "should allow an uppercase short form" do
+      option = Puppet::Interface::Option.new(face, "-F json", "--format json")
```

Comment:
Is there an easy way to validate that passing the face -F actually results in `:format` being set?